README.md

```diff
@@ -67,24 +67,26 @@ configure your accessory using JSON:
   "ipaddress": "192.168.x.xxx",
   "dockContactSensor": true,
   "runningContactSensor": true,
-  "binContactSensor": true
+  "binContactSensor": true,
+  "dockOnStop": true
 }
 ```
 
-|Key|Description|
-|---|-----------|
-|`accessory`|Loads this plugin. Must be set to `Roomba2`|
-|`name`|The name of your Roomb as it should appear in Homebridge and HomeKit|
-|`model`|The model of your Roomba as you'd like it to appear in HomeKit|
-|`serialnum`|The serial number as you'd like it to appear in HomeKit|
-|`blid`|The `blid` of your Roomba, obtained during setup|
-|`robotpwd`|The password for your Roomba, obtained during setup|
-|`ipaddress`|The IP address of your Roomba on your network|
-|`dockContactSensor`|Add a contact sensor to HomeKit that's _closed_ when Roomba is docked|
-|`runningContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba is running|
-|`binContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba's bin is full|
-|`dockingContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba is docking|
-|`noDockOnStop`|Do not send home when stopped|
+
+|Key|Description|Default Value|
+|---|-----------|---|
+|`accessory`|Loads this plugin. Must be set to `Roomba2`||
+|`name`|The name of your Roomba as it should appear in Homebridge and HomeKit||
+|`model`|The model of your Roomba as you'd like it to appear in HomeKit||
+|`serialnum`|The serial number as you'd like it to appear in HomeKit||
+|`blid`|The `blid` of your Roomba, obtained during setup||
+|`robotpwd`|The password for your Roomba, obtained during setup||
+|`ipaddress`|The [IP address](#troubleshooting) of your Roomba on your network||
+|`dockContactSensor`|Add a contact sensor to HomeKit that's _closed_ when Roomba is docked|`true`|
+|`runningContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba is running||
```

Comment:
This must default to true or false, as do the other contact sensors, as it either appears or doesn't if it isn't set. I think it defaults false? Please check, as I'm going by memory while remote. And could you add defaults for those sensors to the config schema too?